demo-shell/src/app/components/cloud/people-groups-cloud-demo.component.ts

```diff
@@ -117,6 +120,25 @@ export class PeopleGroupCloudDemoComponent {
     onChangePeopleValidation(event: MatCheckboxChange) {
         this.peoplePreselectValidation = event.checked;
         this.preSelectUsers = [...this.preSelectUsers];
+        if (!this.peoplePreselectValidation) {
+            this.invalidUsers = [];
+        }
+    }
+
+    onChangeGroupValidation(event: MatCheckboxChange) {
+        this.groupPreselectValidation = event.checked;
+        this.preSelectGroup = [...this.preSelectGroup];
+        if (!this.groupPreselectValidation) {
+            this.invalidGroups = [];
+        }
+    }
+
+    onGroupsWarning(warning: any) {
+        this.invalidGroups = warning.groups;
```

Comment:
Can the any type change to something else? We always emit a specific type of warning which is `this.warning.emit({
                message: 'INVALID_PRESELECTED_GROUPS',
                groups: this.invalidGroups
            });`